bindings/python/test/test_schema.py

```diff
@@ -14,12 +14,36 @@
 from datetime import datetime
 from unittest import TestCase
 
-from bson import Int64
-from pyarrow import float64, int64, timestamp
+from bson import Decimal128, Int64, ObjectId
+from pyarrow import Table, float64, int64
+from pyarrow import schema as ArrowSchema
+from pyarrow import timestamp
 from pymongoarrow.schema import Schema
+from pymongoarrow.types import _TYPE_NORMALIZER_FACTORY
 
 
 class TestSchema(TestCase):
+    def test_as_py(self):
```

Comment:
I'm having trouble following along with this test.  I think some comments and more descriptive variable names are needed.  Also, it looks like you're using `_id` for all of the dtypes, perhaps that should be called `value`?